DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/AddSupervisorSummary.cshtml

```diff
@@ -27,7 +28,7 @@
       Supervisor
     </dt>
     <dd class="nhsuk-summary-list__value">
-      @Model.Supervisor.Forename @Model.Supervisor.Surname (@Model.Supervisor.Email)
+      @DisplayStringHelper.GetPotentiallyInactiveAdminName(Model.Supervisor.Forename, Model.Supervisor.Surname, Model.Supervisor.Active) (@Model.Supervisor.Email)
```

Comment:
The data service that retrieved possible supervisors to add includes `WHERE Active = 1` so this helper method is unlikely to be needed, though I suppose it's not doing any harm.